### PR TITLE
add Eudora

### LINKED ISSUE #1627: Eudora
It seems like [Eudora](https://wiki.mozilla.org/Eudora_Releases) is not available.
Here is the user-agent, i found:

<pre>
Eudora/6.2.4b8 (MacOS)
Mozilla/5.0 (Windows; U; Windows NT 6.1; en-US; rv:1.9.1.9) Gecko/20100722 Eudora/3.0.4
Mozilla/5.0 (Windows; U; Windows NT 6.2; en-US; rv:1.9.1.9) Gecko/20100722 Eudora/3.0.4
</pre>

### PR DESCRIPTION
**Pull request by mimmi20**
Fixes #1627.

### CODE REVIEW

File: resources/user-agents/email-clients/eudora/eudora.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    {
+      "userAgent": "Eudora OSE",
+      "platform": "Windows",
+      "engine": "Gecko",
+      "device": "Windows Desktop",
+      "properties": {
+        "Parent": "DefaultProperties",
+        "Comment": "Eudora OSE",
+        "Browser": "Eudora OSE",
```
Comment (jaydiablo) [inline resources/user-agents/email-clients/eudora/eudora.json]: What is "OSE"?

Comment (mimmi20) [reply resources/user-agents/email-clients/eudora/eudora.json]: Open Source Edition

Comment (mimmi20) [reply resources/user-agents/email-clients/eudora/eudora.json]: Shall I use the full name?

Comment (jaydiablo) [reply resources/user-agents/email-clients/eudora/eudora.json]: Ah, maybe in the Comment?

Also, it seems that perhaps Eudora OSE also ran/runs on OSX and Linux based on this wiki, though I can't find any example UAs in our logs.  https://wiki.mozilla.org/Eudora_OSE

**Pull request merged**